packages/message-utils/README.md

```diff
@@ -0,0 +1,23 @@
+[![License][badge-license]][license]
+[![Version][badge-version]][package]
+[![Downloads][badge-downloads]][package]
+
+# @lingui/message-utils
+
+> Internal package. You probably don't need to use it directly.
```

Comment:
Maybe it's a good idea to describe what functionality this package provides and why?

Reply:
I don't have ideas what to write there. More over it's internal package and i don't want to someone use it directly